tee-worker/ts-tests/identity.test.ts

```diff
@@ -368,7 +368,7 @@ describeLitentry('Test Identity', 0, (context) => {
             context.substrateWallet.bob,
             bob_txs,
             'identityManagement',
-            ['IdentityVerified']
+            ['IdentityVerified', 'VerifyIdentityFailed'] // allow possible failed events to be collected
         );
         const verified_event_datas = await handleIdentityEvents(context, aesKey, alice_resp_events, 'IdentityVerified');
```

Comment:
> 


I found that the description of #1684 may be incorrect. If you continue testing, you will eventually encounter a `timeout`. The console only[ prints the current event](https://github.com/litentry/litentry-parachain/blob/dev/tee-worker/ts-tests/common/transactions.ts#L106) and its content without any further meaning, and the "ts-test" will not succeed [here](https://github.com/litentry/litentry-parachain/blob/dev/tee-worker/ts-tests/common/transactions.ts#L163). Therefore, it will eventually timeout. This [pr](https://github.com/litentry/litentry-parachain/pull/1624) addresses the optimization related to the timeout issue.

> In case of `VerifyIdentityFailed` shouldn't `sendTxsWithUtility` timeout ?
> 
> ```
> await sendTxsWithUtility(
>             context,
>             context.substrateWallet.bob,
>             bob_txs,
>             'identityManagement',
>             ['IdentityVerified']
> ```
> 
> @Verin1005 ?

I found that the description of #1684 may be incorrect. If you continue testing, you will eventually encounter a `timeout`. The console only[ prints the current event](https://github.com/litentry/litentry-parachain/blob/dev/tee-worker/ts-tests/common/transactions.ts#L106) and its content without any further meaning, and the "ts-test" will not succeed [here](https://github.com/litentry/litentry-parachain/blob/dev/tee-worker/ts-tests/common/transactions.ts#L163). Therefore, it will eventually timeout. This [pr](https://github.com/litentry/litentry-parachain/pull/1624) addresses the optimization related to the timeout issue.